homeassistant/components/aten_pe/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "aten_pe",
+  "name": "ATEN eco PDUs",
+  "documentation": "https://www.home-assistant.io/components/aten_pe",
+  "requirements": [
+    "atenpdu==0.1.2",
+    "pysnmp==4.4.11"
```

Comment:
This requirement is not directly used by this integration.
Seems like `atenpdu` already depends on it, so there should be no need to add it here.